### ISSUE TITLE
Wishlist: Render file management

### ISSUE THREAD

**Issue by hiroshisasak1** [jungmannlab/picasso#33]
I'm wondering if Render could have the "Save all at once" option for "Save localizations". This option is currently available only for "Save picked localizations".

It would be also helpful if we could close a localization file opened in Render. I often open multiple files in the same Render window for aligning channels, and I have to start all over when I open a wrong file or the channel alignment doesn't work.

**Comment by straussmaximilian** [comment]
Hi, that's a great idea. I just implemented the "Save all at once for the files" (commit 1177). Additionally, I implemented a dialog to set the suffix (for files and pick props) to avoid unintentional overwriting.

As for the closing, I thought about including this in the dataset/file dialog where the currently displayed files are listed. For now, I added some buttons (which are without function for now). Unfortunately, the internal file management is a bit messy right now, and I'll first need to wrap my head around on how to remove the loaded files best.
